### PR TITLE
Syslog Docker

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Status
Ready

## Related Content Pull Request
Related PR: https://github.com/demisto/content/pull/15716

## Related Issues
Related: [link to the issue](https://github.com/demisto/etc/issues/28637)

## Description
Added a docker image to contain Syslog format parsers

### CODE REVIEW

File: docker/syslog/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.9.7.24076
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
+  libffi-dev openssl-dev \
```
Comment (dantavori) [inline docker/syslog/Dockerfile:7]:
```suggestion
  libffi-dev \
```

File: docker/syslog/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+FROM demisto/python3:3.9.7.24076
```
Comment (dantavori) [inline docker/syslog/Dockerfile:2]:
```suggestion
FROM demisto/python3:3.9.8.24399
```

**Review by dantavori — CHANGES_REQUESTED** [review]
as discussed, please use the new `add_dependabot.sh` script

File: docker/syslog/Dockerfile (hunk 1 above)
Comment (tomneeman151293) [reply docker/syslog/Dockerfile:7]: openssl is actually needed, tagged u in the slack thread regarding it

Comment (dantavori) [reply docker/syslog/Dockerfile:7]: @wissamg fyi

**Review by dantavori — APPROVED** [review]

**Pull request merged**